cmd/rest/main.go

```diff
@@ -36,10 +36,12 @@ func Run() error {
 	db, err := pg.ConnectDB(config.DB)
 	if err != nil {
 		logger.Errorf("%+v", err)
-		return err
+
+		return fmt.Errorf("error connecting to database: %w", err)
```

Comment:
You might want to add some details about the `config.DB` into this error.

Reply:
done